lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -256,6 +256,7 @@ def populate_api_routes( webapp, app ):
     webapp.mapper.resource( 'remote_file', 'remote_files', path_prefix='/api' )
     webapp.mapper.resource( 'group', 'groups', path_prefix='/api' )
     webapp.mapper.resource_with_deleted( 'quota', 'quotas', path_prefix='/api' )
+    webapp.mapper.connect( '/api/interactive_environment_ready', controller='interactive_environment_ready' )
```

Comment:
Planning for the future (since this is the API and all), maybe go with something ```GET /api/interactive_environment/<key>/ready```?